README.md

```diff
@@ -1,137 +1,113 @@
 # sklearn-numba-dpex
 
-Experimental plugin for scikit-learn to be able to run (some estimators) on
-Intel GPUs via [numba-dpex](https://github.com/IntelPython/numba-dpex).
+Experimental plugin for scikit-learn to be able to run (some estimators) on Intel GPUs
+via [numba-dpex](https://github.com/IntelPython/numba-dpex). Support for other GPU
+constructors is also on the roadmap and depends on progress of interoperability features
+of the `numba-dpex` stack.
 
-DISCLAIMER: this is work in progres, do not expect this repo to be in a
-workable state at the moment.
-
-This requires working with the following branch of scikit-learn which is itself
-not yet in a working state at the moment:
+This package requires working with the following branch of scikit-learn:
 
 - `wip-engines` branch on https://github.com/ogrisel/scikit-learn
 
 ## List of Included Engines
 
-- `sklearn.cluster.KMeans` for the standard LLoyd's algorithm on dense data arrays.
-  The k-means++ initialization scheme is not implemented yet but this is planned.
+- `sklearn.cluster.KMeans` for the standard LLoyd's algorithm on dense data arrays,
+  including `kmeans++` support.
 
   No implementation of k-means for sparse input data or for the "Elkan" algorithm is
-  included at the moment.
+  included at the moment or planned to be.
```

Comment:
I think it might be more interesting for users to know what is planned to be supported rather than what isn't. What do you think?